src/components/EditSciApplication.vue

```diff
@@ -579,8 +588,13 @@ export default {
         pi_institution: initialSciApp.pi_institution || '',
         coinvestigator_set: initialSciApp.coinvestigator_set || [],
         timerequest_set: initialSciApp.timerequest_set || [],
+        tags: initialSciApp.tags || [],
         pdf: null
       };
+      let isStudent = false;
+      if (sciApp.tags.indexOf('student') >= 0) {
```

Comment:
why not just `sciApp.tags.includes('student')`?

Reply:
Done